.eslintrc.json

```diff
@@ -15,6 +15,10 @@
     }
   },
   "rules": {
+    "max-len": [
+      "error",
+      { "ignoreTemplateLiterals": true, "ignoreUrls": true }
```

Comment:
What about `{"code": 80, "ignore...` just to be extra explicit?

And are the padding spaces after `{` and before `}` needed?

Reply:
@jorgeorpinel We can add it but it's by default 80.
The bracket-spacing is prettier's [default behaviour](https://prettier.io/docs/en/options.html#bracket-spacing). We can disable it though.

Reply:
@jorgeorpinel trust you judgment on this one. I'm totally fine with this.

Reply:
Well, since we specifically talk about 80 char lines in https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-and-tips-for-java-script-and-markdown, I think we should make it explicit here. Please add it, @algomaster99. Thanks.